main.go

```diff
@@ -108,7 +113,12 @@ func main() {
 		templates.SetProxiesURL(proxiesURL)
 	}
 
-	go utils.CheckCountry([]string{"Ukraine"})
+	countries := strings.Split(countryList, ",")
+	if !utils.CheckCountry(countries) {
+		if strictCountryCheck {
+			return
```

Comment:
Probably worth adding some logs here to avoid confusion

Reply:
Done